app/views/avo/partials/_keep_modal_open.turbo_stream.erb

```diff
@@ -0,0 +1,3 @@
+<%= turbo_stream.append "alerts" do %>
+  <%= render Avo::FlashAlertsComponent.new flashes: flash %>
+<% end %>
```

Comment:
Maybe we should rename this stream to `.../_flash_alerts.turbo_stream.erb` i think it fits better to the file porpoise.